docs/site/Discovering-models.md

```diff
@@ -49,7 +49,10 @@ placed. Default is `src/models`
 `--schema`: Specify the schema which the datasource will find the models to
 discover
 
+**`--models`**: Specify the models to be generated
```

Comment:
it would be good to mention the format `--models=table1,table2` as you listed in the code.